config/kubernetes/sig-docs/teams.yaml

```
@@ -1,4 +1,11 @@
 teams:
+  sig-docs-blog-owners:
+    description: Approvers for blog content
+    members:
+    - mrbobbytables
+    - nate-double-u 
+    - onlydole
+    - sftim
```

please set `privacy: closed` as well.
```suggestion
    - sftim
    privacy: closed
```

---

I do smile every time I see `privacy: closed` like this in public source code.